src/main/kotlin/no/nav/faktureringskomponenten/service/FakturaserieService.kt

```diff
@@ -21,8 +21,8 @@ class FakturaserieService(
             message = "Fant ikke fakturaserie på: $referanse"
         )
 
-    fun hentFakturaserier(referanse: String, fakturaStatus: String?): List<Fakturaserie> {
-        return fakturaserieRepository.findAllByReferanse(referanse, fakturaStatus)
+    fun hentFakturaserier(referanse: String): List<Fakturaserie> {
+        return fakturaserieRepository.findAllByReferanse(referanse)
```

Comment:
Man kan fortsatt filtrere her og ikke nødvendigvis i frontend, siden det som ønskes er å forenkle repository-koden

Reply:
Jeg mangler noe context her mtp filtrering. Men om det er mange flere variabler enn fakturastatus, tenker jeg det er like greit å bare sende data ufiltrert til frontend og heller håndtere det der.

Reply:
Jeg tenkte som Øystein nevnte under her, at man kan heller sende en ufiltrert liste ut og heller gjøre filtreringen et annet sted. Da har denne funksjonen en oppgave, det er bare å hente fakturaserier som hører til den referansen som kommer inn.

Reply:
Greit for meg hvis dette kan brukes i frontend. Blir uansett ikke en stor datamengde

Reply:
Mhm, jeg tenkte det, datamengden er liten